concepts/strings/links.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
https://go101.org/article/string.html is not too bad if you are looking for a decent reference to give

Reply:
@junedev What do you think about the above link? It does mention runes and slices.

Reply:
I think more advanced articles are ok for the links as this is labelled as "Read More" on the website.

Reply:
👍 I'll add it